carbonmark-api/src/routes/projects/get.utils.ts

```diff
@@ -62,36 +59,52 @@ export const getDefaultQueryArgs = async (
  * Chooses between default and selective price.
  * Sorted cheapest first.
  */
-export const getOffsetTokenPrices = (
-  offset: GetCarbonOffsetsByProjectAndVintageQuery["carbonOffsets"][number],
+type ProjectBalances = Record<string, string>;
+
+export const getDigitalCarbonTokenPrices = (
+  digitalCarbonProject: FindQueryDigitalCarbon,
   poolPrices: Record<string, PoolPrice>
 ) => {
+  const pools = digitalCarbonProject.carbonCredits[0].poolBalances;
+
+  const tokenBalances: ProjectBalances = {};
+
+  for (const pool of pools) {
+    const poolInfoEntries = Object.entries(POOL_INFO);
+    for (const [key, poolInfo] of poolInfoEntries) {
+      if (poolInfo.poolAddress === pool.id) {
+        tokenBalances[key] = pool.balance;
+      }
+    }
+  }
```

Comment:
This is a little confusing.. could you add a comment as to what we're doing here? looks like we're extracting the balance for each possible token?